src/pages/_app.tsx

```diff
@@ -3,16 +3,18 @@ import type { AppProps } from 'next/app';
 import { ToastContainer } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
 import '@/styles/index.scss';
+import AppWrapper from '@/context'
```

Comment:
Wouldn't a better name would be `AppContextWrapper` or something. You are wrapping the app with _something_ here.